### DISCUSSION TITLE
How do we preview the documentation?

### DISCUSSION THREAD

**Discussion by colinkiama** [elementary/docs#219] [Q&A]
There are some GitBook-specifc blocks such as  [the Tabs Block](https://docs.gitbook.com/content-editor/blocks/tabs).

We're not able to preview these with standard markdown previews available in editor plugins and online.

If there isn't a solution, I think it may be worth looking for an alternative project documentation solution for elementary.

**Comment by danirabbit** [comment]
Yeah unfortunately I think the only way is through the gitbook.io interface. I'm not opposed to switching to another documentation system that can be built locally. We switched to Gitbook a while ago because we had our own documentation system which was not good, and Gitbook was well-maintained, feature rich, and nice looking
